torch/csrc/WindowsTorchApiMacro.h

```diff
@@ -1,9 +1,2 @@
 #pragma once
-
-#include <c10/macros/Export.h>
-
-#ifdef _WIN32
-#define TORCH_PYTHON_API
-#else
-#define TORCH_PYTHON_API TORCH_API
-#endif
+#include <torch/csrc/Export.h>
```

Comment:
Should we just delete this file? (can be in a follow up if you prefer)

Reply:
We should, but it's included everywhere in the torch folder so the diff is massive. Can do it in separate PR though.

Reply:
Follow up PR sounds good!